change/office-ui-fabric-react-24a1d857-4f53-43b4-b031-0586daed6ca6.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Ported viewport fix to 7.0",
```

nit: It should probably say the specific fix.

```suggestion
  "comment": "Porting fix that delays calculating viewport on mount to 7.0",
```